tests/src/JIT/HardwareIntrinsics/X86/Shared/SimdScalarUnOpConvTest.template

```diff
@@ -141,7 +141,7 @@ namespace JIT.HardwareIntrinsics.X86
             _dataTable = new SimdScalarUnaryOpTest__DataTable<{Op1BaseType}>(_data, LargestVectorSize);
         }
 
-        public bool IsSupported => {Isa}.IsSupported && (Environment.Is64BitProcess || ((typeof({RetBaseType}) != typeof(long)) && (typeof({RetBaseType}) != typeof(ulong))));
```

Comment:
Did we update all the templates that had this check?

Reply:
Ah, a few templates still have this check. Can I update it in the next PR?

Reply:
> Can I update it in the next PR?

And will move more tests to the template framework.

Reply:
Yes, that would be great. Thanks!